views/files/files.hbs

```diff
@@ -57,8 +57,7 @@
 				<div class="row">
 					{{#each directories}}
 						<div class="col-xs-12">
-						  <div class="card card-block folder openfolder" data-folder-id="{{this._id}}">
-                              <button type="button" class="button-transparent">
+						  <div class="card card-block folder openfolder" data-folder-id="{{this._id}}" tabindex="0" role="button" aria-label="{{$t "files.label.clickToOpenDir" (dict "name" this.name)}}">
```

Comment:
better to make it a button and restyle it with css

Reply:
done